src/components/DocumentSelector/index.js

```diff
@@ -29,11 +29,11 @@ class DocumentSelector extends Component<Props & WithDefaultOptions> {
 
   getOptions = () => {
     const {i18n, documentTypes, defaultOptions, country = 'GBR' } = this.props
-    const defaultDocOptions = defaultOptions(i18n)
+    const defaultDocOptions = defaultOptions(i18n).filter(
+      ({ checkAvailableInCountry = always }) => checkAvailableInCountry(country)
```

Comment:
Will this affect the filtering of ID docs?

Reply:
It will, but since no ID doc defines a `checkAvailabilityInCountry`, they will always pass:

https://github.com/onfido/onfido-sdk-ui/blob/development/src/components/DocumentSelector/documentTypes.js#L17